ddmrp/models/procurement_order.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# © 2016 Aleph Objects, Inc. (https://www.alephobjects.com/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    add_to_net_flow_equation = fields.Boolean(
+        string='Pending add net flow equation', default=True,
+        readonly=True,
+        help="If this flag is set, the procurement is pending to be included "
+             "into the net flow equation.")
+
+    @api.multi
+    def write(self, vals):
+        res = super(ProcurementOrder, self).write(vals)
+        if 'state' in vals.keys() \
```

Comment:
This condition could maybe be written as 

`if vals.get('state', False) not in ['draft', 'cancel'] and not vals.get('add_to_net_flow_equation', False):`

It covers the case (I don't know how likely) that 'add_to_net_flow_equation' is explicitly passed as False in the method arguments) and, IMHO is also more readable.